### DISCUSSION TITLE
Broadcast screen remains on client even after attempted 'unbroadcasting'

### DISCUSSION THREAD

**Discussion by dgroos** [epoptes/epoptes#34] [Q&A]
At the start of class I select all clients in epoptes, then broadcast my desktop with them so they can better see the screen when I present something at the start of class.

However, sometimes a student or 2 might be logged in by the time I initiate broadcast to them. Broadcast works fine but when it comes time to un-broadcast my screen, all clients that hadn't logged in work as expected, but I am unable to un-broadcast to the pre-logged-in client. While other students start logging in, this unfortunate student remains stuck. To solve this I log them out through epoptes upon which they can log back in.

**Comment by dgroos** [comment]
Problem continues...
This problem is completely reproducibly. Here's a ss of a "Root, locally" terminal from epoptes showing the syslog from when there was an instance of this problem.

![clients-lock-up](https://user-images.githubusercontent.com/4097759/37492857-e7b7d044-2870-11e8-8f3c-9903273d161c.png)

**Comment by alkisg** [comment]
The older 0.5.10 Epoptes version is based on Python2/Gtk2 and will not receive any updates.

I uploaded the newer 1.0.0 version in the [Epoptes proposed PPA](http://www.epoptes.org/documentation/ppa) for Ubuntu 18.10, 18.04 and 16.04.
If you still have a the problem with that version, I could take a look.

**Comment by dgroos** [comment]
No longer a problem, Thanks!
